spec/lob/v1/address_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 describe Lob::V1::Address do
 
-  let(:sample_params) {
-    {
+  before :each do
+    @sample_params = {
```

Comment:
Ah okay. I also [prefer](https://stackoverflow.com/questions/5359558/when-to-use-rspec-let/5359979#5359979) `let`, but consistency is 💯